android-o/EmojiCompat/README.md

```diff
@@ -27,7 +27,7 @@ Using this sample requires the Android 8.0 (API 26) and the Xamarin.Android 7.5
 Screenshots
 -------------
 
-<img src="screenshots/Main.png" height="400" alt="Screenshot"/> 
+<img src=“Screenshots/Main.png" height="400" alt="Screenshot"/> 
```

Comment:
Actually, that section (Screenshot) was copied from original README and I removed it. It is not in current README file. If a screenshot link is needed, I can add it as you mentioned.